templates/components/api-server/template/server/server.js

```diff
@@ -1,7 +1,11 @@
 var loopback = require('loopback');
 var boot = require('loopback-boot');
+var helmet = require('helmet');
 
 var app = module.exports = loopback();
+//use helmet to improve app security: more details on Best practices for Express in Production:
+// https://strongloop.com/strongblog/best-practices-for-express-in-production-part-one-security/
+app.use(helmet());
```

Comment:
All middleware should be configured via `server/middleware.json`. To make it easier for users to tweak helmet config, the scaffolded `server/middleware.json` should include configuration mirroring helmet's defaults. That way users can just change a flag value, instead of having to look up all flag names in helmet docs.